### ISSUE TITLE
data.google_organization.id returns organization/id instead of just the id

### ISSUE THREAD

**Issue by Nosmoht** [terraform-providers/terraform-provider-google#5098]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment
* If an issue is assigned to the "modular-magician" user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If an issue is assigned to a user, that user is claiming responsibility for the issue. If an issue is assigned to "hashibot", a community member has claimed the issue already.

### Affected Resource(s)

* data.google_organization

### Terraform Configuration Files

```tf
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://www.hashicorp.com/security
# If reproducing the bug involves modifying the config file (e.g., apply a config,
# change a value, apply the config again, see the bug) then please include both the
# version of the config before the change, and the version of the config after the change.
provider "google" {
  version = "= 3.1.0"
}

data "google_organization" "test" {
  domain = "example.com" # << please change to an org your account has permissions to
}

output "test" {
  value = data.google_organization.test
}
```

### Debug Output

```tf
test = {
  "create_time" = "2019-08-09T12:14:48.112Z"
  ...
  "id" = "organizations/1234567890"
  "name" = "organizations/1234567890"
  ...
}
```

### Expected Behavior
data.google_organization.id should just return the `id`, not `organizations/id`.

### Actual Behavior
data.google_organization.id returns `organization/id`.

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by Chupaka** [comment]
https://www.terraform.io/docs/providers/google/guides/version_3_upgrade.html#id-format-changes

**Comment by Nosmoht** [comment]
Wow, that was fast. Anyhow docu just says
```
The following additional attributes are exported:

id - The Organization ID.
name - The resource name of the Organization in the form organizations/{organization_id}.
```
https://www.terraform.io/docs/providers/google/d/google_organization.html

**Comment by Nosmoht** [comment]
Just for others, my current workaround
```tf
org_id  = tonumber(replace(tostring(data.google_organization.company.id), "organizations/", ""))
```

**Comment by Chupaka** [comment]
I believe, the problem is "id" was not officially exported anywhere with a few exceptions like this one, so it has to be addresses personally.

**Comment by Nosmoht** [comment]
Honestly, this has a bigger impact. So far for `google_project_service` it tells me that replacement while running `terraform plan` would be forced and results in an error while running `terraform apply`

terraform plan output
``` 
  # module.cicd.google_project_service.container["dev"] must be replaced
-/+ resource "google_project_service" "container" {
      + disable_on_destroy = true
      ~ id                 = "cicd-dev/container.googleapis.com" -> (known after apply)
      ~ project            = "cicd-dev" -> "projects/cicd-dev" # forces replacement
        service            = "container.googleapis.com"

      - timeouts {}
    }
```

terraform apply output
```
Error: failed to send enable services request: googleapi: got HTTP response code 404 with body: <!DOCTYPE html>
<html lang=en>
  <meta charset=utf-8>
  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
  <title>Error 404 (Not Found)!!1</title>
  <style>
    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}
  </style>
  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>
  <p><b>404.</b> <ins>That’s an error.</ins>
  <p>The requested URL <code>/v1/projects/projects/cicd-dev/services/container.googleapis.com:enable?alt=json&prettyPrint=false</code> was not found on this server.  <ins>That’s all we know.</ins>


  on .terraform/modules/cicd/projects.tf line 37, in resource "google_project_service" "container":
  37: resource "google_project_service" "container" {
```

**Comment by neomantra** [comment]
So I was bit by this... started working on a PR to add `org_id` and see that this change was [committed  two days ago](https://github.com/terraform-providers/terraform-provider-google/commit/8baff6ea14b32f414289db2a13f5706219fcdb0c)

Once this is released, you can use `data.google_organization.name.org_id`

You can build your own latest with [these instructions](https://github.com/terraform-providers/terraform-provider-google#building-the-provider).

But I'm using @Nosmoht 's snippet for now

**Comment by neomantra** [comment]
This `org_id` commit has landed on `v3.2.0`

**Comment by rileykarson** [comment]
Sorry about the disruption! This and a few resources- particularly, datasources- slipped through the cracks and didn't have a great equivalent. `org_id` supersedes old uses of `id`, which held the short-form ID value (`012345678910`), and `id` across most resources is expected to hold the long-form value (`organizations/012345678910`). 

I'm closing this out since that's available, if I missed anything or you feel this isn't resolved, feel free to reopen it / comment!
